src/current/v23.2/known-limitations.md

```diff
@@ -10,100 +10,61 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
+  - UDF and stored procedure definitions do not support `OUT` and `INOUT` argument modes. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/100405)
 
-[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/92961)
+### SQL Optimizer and Read Committed Isolation
 
-### Low estimated Request Units are rounded to zero
+#### Optimizer and Locking Behavior
 
-The [Request Units](https://www.cockroachlabs.com/docs/cockroachcloud/plan-your-cluster-serverless#request-units) (RUs) estimate surfaced in [`EXPLAIN ANALYZE`]({% link {{ page.version.version }}/explain-analyze.md %}) is displayed as an integer value. Because of this, fractional RU estimates, which represent very inexpensive queries, are rounded down to zero.
+- The SQL optimizer has limitations under certain isolation levels:
+  - The new implementation of `SELECT FOR UPDATE` is not yet the default setting under `SERIALIZABLE` isolation. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/114737)
+  - `SELECT FOR UPDATE` does not lock completely-`NULL` column families in multi-column-family tables. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/116836)
+  - `SELECT ... FOR UPDATE` and `SELECT ... FOR SHARE` locks are dropped on lease transfers and range splits/merges. Unreplicated locks, used by default under SERIALIZABLE isolation, can be dropped during such events, affecting the desired ordering of concurrent accesses and potentially leading to transaction retry errors. [More information on `SELECT ... FOR UPDATE`](https://www.cockroachlabs.com/docs/v23.2/select-for-update)
 
-[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/100617)
+#### Read Committed Isolation Limitations
 
-### Statistics for deleted tables in `system.table_statistics` do not get removed
+- Several capabilities are not yet supported with [Read Committed isolation](https://www.cockroachlabs.com/docs/v23.2/read-committed):
+  - DDL statements cannot be performed within explicit Read Committed transactions. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/114778)
+  - Specific unique checks are not supported under Read Committed isolation. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/110873)
+  - `SKIP LOCKED` requests do not reliably skip over replicated locks. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/110743)
+  - Promoting a lock from `shared` to `exclusive` is not supported. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/110435)
+  - Schema changes are not allowed within explicit transactions, and certain unique constraints are not supported. Shared locks cannot be promoted to exclusive locks, and `SKIP LOCKED` requests do not reliably skip over replicated locks. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/115057)
```

Comment:
This item seems like a repeat of several items above, squashed together. I think it can be removed.